api/songs/respository/song-repository.ts

```diff
@@ -231,15 +231,39 @@ export const createSongRepository = (client: DynamoDB) => {
       },
       UpdateExpression: "SET voteCount = :voteCount REMOVE GSI2PK, voters",
     });
-
     // throw "NYI: need to remove votes";
   };
 
+  const addLockToSong = async (id: string): Promise<void> => {
+    await client.updateItem({
+      TableName: "song",
+      Key: {
+        PK: {
+          S: id,
+        },
+        SK: {
+          S: id,
+        },
+      },
+      ReturnValues: "UPDATED_NEW",
+      ExpressionAttributeValues: {
+        ":zero": {
+          N: "0",
+        },
+        ":playlist": {
+          S: "ON_PLAYLIST",
+        },
+      },
+      UpdateExpression: "SET lockOrder = 1, GSI2PK = :playlist",
```

Comment:
Dynamo forces you to use parameterized values, so you can't do `SET lockOrder = 1`. You need to do something like `SET lockOrder = :count` and then bind `:count`. That's why I need to do this weird syntax 
```
        ":zero": {
          N: "0",
        },
```
and then bind it in the `UpdateExpression` field. 

Should work fine then.

Reply:
I've changed this to `:one`. Every `lockOrder` will be set to 1 for now.